Surface_mesh/include/CGAL/Surface_mesh/Surface_mesh.h

```diff
@@ -44,6 +45,7 @@
 #include <CGAL/boost/graph/graph_traits_Surface_mesh.h>
 #include <CGAL/boost/graph/iterator.h>
 #include <CGAL/boost/graph/Euler_operations.h>
+#include <CGAL/IO/File_scanner_OFF.h>
 
```

Comment:
#include <CGAL/IO/File_scanner_OFF.h> is needed to acces the functions you made me delete from Surface-mesh.h because they were already in File_scanner_OFF.h.